docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
+
+Additionally, the event history for Free Tier workflows is limited to only the past 100 events.
+
+Upgrading to a paid tier will increase this event history tracking substantially.
+
+### Free Tier Polling Interval Limitations
+
+Free Tier account triggers powered by polling are limited to 5 minute intervals. Paid tiers have an option to polling at a substantially higher frequency. 
+
+### Free Tier Support options
 
 Users on the Developer Tier have access to community support, on [our forum](https://pipedream.com/community) and Slack. [Visit out Support page](https://pipedream.com/support) for more information.
 
-## Professional Tier
+## Paid Tiers
 
-The Professional Tier includes all the features of the Developer Tier. It also comes with the following benefits:
+[Visit our pricing page](https://pipedream.com/pricing) to learn more about our paid plan options.
 
-- **You have no daily invocations or compute time cap. You can run any number of invocations, for any amount of time**.
-- Your event history per workflow increases from {{$site.themeConfig.FREE_INSPECTOR_EVENT_LIMIT}} to {{$site.themeConfig.PAID_INSPECTOR_EVENT_LIMIT}}.
-- You can increase the [max time per execution](/limits/#time-per-execution) to 750 seconds (up from 300 on the Developer Tier).
-- You can request [QPS increases](/limits/#qps-queries-per-second) for specific HTTP endpoints.
-- When using [concurrency and throttling controls](/workflows/concurrency-and-throttling/), you can increase a workflow's queue size up to {{$site.themeConfig.MAX_WORKFLOW_QUEUE_SIZE}} (free users are capped at a queue size of {{$site.themeConfig.DEFAULT_WORKFLOW_QUEUE_SIZE}}).
+All paid plans vary features based on tier, but each paid plan option will:
 
-### Upgrading to the Professional Tier
+* Lift the daily 100 credits limit
+* Increase the number of active workflows available
+* Increase the number of connected accounts
 
-You can upgrade to the Professional Plan by visiting your [Billing and Usage Settings](https://pipedream.com/settings/billing).
+## Definition of Terms
 
-### Professional Tier Pricing
+Pipedream uses a number of terms to describe platform metrics and details of our plans. See the definitions of key terms below.
 
-See [https://pipedream.com/pricing](https://pipedream.com/pricing) for pricing information.
+### Credits
 
-### Professional Tier Support options
+::: tip
 
-Users on the Professional Tier have access to community support, on [our forum](https://pipedream.com/community) and Slack. [Visit out Support page](https://pipedream.com/support) for more information.
+If you're still on our grandfathered Developer, Professional or Organization plan, you still may be incurring **invocations** and not credits.
 
-## Team Plan
+During this transitionary period, you'll still be billed in terms of invocations. However workspace migrations are coming. You'll be guided on how to migrate your account to a [workspace](/docs/workspaces/) in the near future.
```

Comment:
"transitionary" -> "transition"

Reply:
"in terms of invocations" -> "for invocations"

Reply:
"However workspace migrations are coming. You'll be guided on how to migrate your account to a [workspace](/docs/workspaces/) in the near future." -> "We'll be migrating legacy accounts to a credit-based model, and will be reaching out via email to communicate that change."